elliottlib/bzutil.py

```diff
@@ -434,26 +435,54 @@ def get_corresponding_flaw_bugs(tracker_bugs: List[Bug], flaw_bug_tracker,
             list(set(sum([t.corresponding_flaw_bug_ids for t in tracker_bugs], []))),
             verbose=verbose
         )
-        flaw_id_bugs = {bug.id: bug for bug in flaw_bugs}
+        flaw_tracker_map = {bug.id: {'bug': bug, 'trackers': []}
+                            for bug in flaw_bugs}
 
         # Validate that each tracker has a corresponding flaw bug
-        flaw_ids = set(flaw_id_bugs.keys())
-        no_flaws = set()
-        for tracker in tracker_bugs:
-            if not set(tracker.corresponding_flaw_bug_ids).intersection(flaw_ids):
-                no_flaws.add(tracker.id)
-        if no_flaws:
-            msg = f'No flaw bugs could be found for these trackers: {no_flaws}'
+        # and a whiteboard component
+        trackers_with_no_flaws = set()
+        trackers_with_invalid_components = set()
+        for t in tracker_bugs:
+            component = t.whiteboard_component
+            if not component:
+                trackers_with_invalid_components.add(t.id)
+                continue
+
+            # is this component a valid package name in brew?
```

Comment:
This validation will help us make sure component names are accurate, so when
we compare brew names for our logic with data from ProdSec DB - we don't have false alarms